core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/oauth/OAuth2Authenticator.java

```diff
@@ -19,7 +20,9 @@
  */
 public interface OAuth2Authenticator {
 
-    void authorize(String authenticatorUrl, AsyncCallback<OAuthStatus> callback);
+    void authenticate(String authenticatorUrl, AsyncCallback<OAuthStatus> callback);
```

Comment:
first method with authenticatorUrl and second method with authenticationUrl. If there isn't any difference between them let's make them the same